config/runtime.exs

```diff
@@ -20,10 +20,9 @@ import Config
 ##########################
 # General configurations #
 ##########################
-
 config :ueberauth, Ueberauth.Strategy.Google.OAuth,
-  client_id: System.get_env("GOOGLE_CLIENT_ID"),
-  client_secret: System.get_env("GOOGLE_CLIENT_SECRET")
+  client_id: System.get_env("CONFIGURATOR_GOOGLE_CLIENT_ID"),
```

Comment:
We can deal with it once it pops up again (or maybe ignore it for ever), but this likely breaking game client google sign in